doc/content/integrations/cloud-integrations/telemetry2u/index.md

```diff
@@ -0,0 +1,45 @@
+# Telemetry2U
+[Telemetry2U](https://telemetry2u.com/) is a complete IoT platform that requires little set-up and configuration yet still provides advanced features, such as voice alerts, scheduled audit reports, configurable real-time dashboards and even calibrations.
+
+## Prerequisites
+1. [Register user account](https://telemetry2u.com/Identity/Account/Register)  on Telemetry2U and confirm e-mail.
+1. [Activate a 14-day free trial](https://telemetry2u.com/Identity/Account/Manage/Subscriptions) or create a PAYG plan with Telemetry2U.
+1. Devices already added to The Things Stack under an application.
+
+## Add Device to Telemetry2U
+First, [login to your Telemetry2U account](https://telemetry2u.com/Identity/Account/Login) and navigate to __Node Maintenance__ via __Admin__ from the top menu.
+
+Click the __Create New__ link in the top right corner to add a new LoRaWAN device.
+
+Complete the form as required.  The __Description__, __Profile__, __Device EUI__, __Network Provider__ and __Device Configuration__ are the minimum requirements. If your device is currently unsupported leave __Device Configuration__ at the default value.  Click the __Create__ button when complete.
+
+![Add a device in Telemetry2U](add-device.png "Add a device in Telemetry2U")
+
+## Setup The Things Stack Integration
+From your Telemetry2U account, navigate to __Setup Integration__ via __Admin__ from the top menu. Select __The Things Network__ from the __Network Provider__ drop down box and hit the __Add Integration__ button.
+
+![Add an integration](add-integration.png "Add an integration")
+
+Select the __Region__ your TTN account is registered under and make a copy of the __Telemetry2U Token__ with the __Copy to clipboard__ button.
+
+Retrieve your __Application ID__ from The Things Stack and enter it into the __Application ID__ field.
+
+The __API Key__ won't be known until the TTN integration has been created so click the __Save__ button and leave the browser tab open.
+
+![Setup the integration](setup-integration.png "Setup the integration")
```

Comment:
the application ID entered in this screenshot is "ttn-integration", while on screenshots showing TTS you're using the "test" app, so please be consistent there and change this screenshot to show the application ID "test", otherwise users might think that those two things are different